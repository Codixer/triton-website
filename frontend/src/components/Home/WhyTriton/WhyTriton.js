import React, { Component } from "react";
//import styles from "./WhyTriton.scss";
import OpenSansTypography from "../../OpenSansTypography/OpenSansTypography";
import { Typography } from "@rmwc/typography";
import "@material/typography/dist/mdc.typography.css";
import "@material/button/dist/mdc.button.css";
import Container from "../../Container/Container";

class WhyTriton extends Component {
  render() {
    return (
      <Container id="about">
        <OpenSansTypography use="headline4">Why Triton?</OpenSansTypography>
        <p>
          Do you own an international Minecraft server? Would you like to send each player, messages
          in their own language?
          <br />
          Well, look no further! Triton offers this among a whole host of other awesome features!
          This plugin uses a placeholder which will be replaced (dynamically, per player) with a
          message defined in the language file.
        </p>
        <p>
          Triton works with any plugin
          <sup>1</sup> that allows customization of messages.
        </p>
        <Typography use="caption">
          <sup>1 </sup>
          We don't promise it to work with literally all plugins, but most of them should work.
        </Typography>
      </Container>
    );
  }
}

export default WhyTriton;
