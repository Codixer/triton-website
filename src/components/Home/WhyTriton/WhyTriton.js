import React from 'react';

const WhyTriton = ({ content }) => (
  <div>
    <h4>Why Triton?</h4>
    <p>{content}</p>
  </div>
);

export default WhyTriton;
