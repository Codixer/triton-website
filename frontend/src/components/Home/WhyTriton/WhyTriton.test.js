import React from "react";
import WhyTriton from "./WhyTriton";
import renderer from "react-test-renderer";

it("<WhyTriton /> renders correctly", () => {
  const tree = renderer.create(<WhyTriton />).toJSON();
  expect(tree).toMatchSnapshot();
});
