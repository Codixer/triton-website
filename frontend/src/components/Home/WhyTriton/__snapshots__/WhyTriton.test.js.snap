// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WhyTriton /> renders correctly 1`] = `
<div
  id="about"
>
  <span
    className="mdc-typography--headline4"
  >
    Why Triton?
  </span>
  <p>
    Do you own an international Minecraft server? Would you like to send each player, messages in their own language?
    <br />
    Well, look no further! Triton offers this among a whole host of other awesome features! This plugin uses a placeholder which will be replaced (dynamically, per player) with a message defined in the language file.
  </p>
  <p>
    Triton works with any plugin
    <sup>
      1
    </sup>
     that allows customization of messages.
  </p>
  <span
    className="mdc-typography--caption"
  >
    <sup>
      1 
    </sup>
    We don't promise it to work with literally all plugins, but most of them should work.
  </span>
</div>
`;
